src/SFA.DAS.EmployerCommitmentsV2.Web/Mappers/Cohort/CreateCohortRequestMapper.cs

```diff
@@ -21,6 +21,7 @@ public Task<CreateCohortRequest> Map(ApprenticeViewModel source)
                 Email = source.Email,
                 DateOfBirth = source.DateOfBirth.Date,
                 Uln = source.Uln,
+                DeliveryModel = source.DeliveryModel,
```

Comment:
I think we are missing a test from CreateCohortRequestMapperTests

Reply:
This one? 

https://github.com/SkillsFundingAgency/das-employercommitments-v2/blob/29cd88a7ffe75dd4f30dccaa6a1b118aab61141e/src/SFA.DAS.EmployerCommitmentsV2.Web.UnitTests/Mappers/Cohort/CreateCohortRequestMapperTests.cs#L75-L79